src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -186,9 +186,14 @@ public static bool NotUsingHttps(string url)
         [RavenAction("/admin/databases", "PUT", AuthorizationStatus.Operator)]
         public async Task Put()
         {
+            await ServerStore.EnsureNotPassiveAsync();
+            if (ServerStore.IsLeader() == false)
```

Comment:
Won't this break the client API and the studio?
Why shouldn't this work?

Reply:
The problem is, that if you put the database from a non-leader node, you can place a database node on a downed server.
you are correct. I'm trying various solutions... moving this to draft for now